src/WebJobs.Script.WebHost/Scale/TableStorageScaleMetricsRepository.cs

```diff
@@ -303,7 +303,7 @@ internal async Task<IEnumerable<DynamicTableEntity>> ExecuteQuerySafeAsync(Cloud
                     return Enumerable.Empty<DynamicTableEntity>();
                 }
 
-                throw e;
+                throw;
```

Comment:
Should we make these changes to current (3.x) instead?

Reply:
Yes, let me do that.